build.sbt

```diff
@@ -79,5 +102,11 @@ lazy val scalaz = (project in file("scalaz"))
   .settings(moduleName := "github4s-scalaz")
   .settings(buildSettings: _*)
   .settings(scalazDependencies: _*)
-  .dependsOn(github4s)
+  .dependsOn(github4sJVM)
   .enablePlugins(AutomateHeaderPlugin)
+
+lazy val testSettings = Seq(
+  fork in Test := false
+)
+
+addCommandAlias("testAllCoverable", ";docs/test;scalaz/test;github4sJVM/test")
```

Comment:
I did it that way because I was under the impression that scoverage wasn't compatible with scala-js, which was the case until this: https://github.com/scoverage/scalac-scoverage-plugin/pull/118. Do you know which version of the sbt-scoverage plugin are we using on sbt-catalyst-extras? (I haven't been able to find it). Maybe we need to update it for the JS tests to be coverable?